css/global.css

```diff
@@ -1,6 +1,26 @@
 button:has(div):has(div):has(div):has(div):has(svg[aria-label="Split stay"]),
+button:has(div):has(div):has(div):has(div):has(
```

Comment:
The aria label for the split stays button changes depending on the language. There didn't seem to be a clear way to reference this button without affecting other elements, so I manually input the translations. Long term this would be difficult to maintain.